ansible/roles/iroha-docker/tasks/config-gen.yml

```
@@ -52,9 +52,25 @@
         loop: "{{ iroha_all_new_nodes }}"
 
       - name: Generate genesis.block 4
+        set_fact:
+          iroha_genesis_extra_transactions: "{{ iroha_new_genesis_block['block_v1']['payload']['transactions'][1:] | default([]) }}"
+
+      - name: Generate genesis.block 5
```

```suggestion
      - name: Generate genesis.block: combine `iroha_genesis_block_commands` to `block_v1` object
```
Same for other commands. Numbers are less useful than actual comments about what the command does.